cli/internal/workspace/workspace.go

```diff
@@ -0,0 +1,10 @@
+// Package workspace contains some utilities around managing workspaces
+package workspace
+
+import "github.com/vercel/turbo/cli/internal/fs"
+
+// Catalog holds information about each workspace in the monorepo.
+type Catalog struct {
```

Comment:
I think `Info` is a bit generic. I was thinking that maybe in the future, we could make this struct more like a single `map[string]Workspace`, and follow through on https://github.com/vercel/turbo/pull/1229. Does Catalog feel more intuitive with that in mind? (`workspace.Group` or `Workspaces` also didn't feel good enough to me). Of course, this abstraction might not actually go any further happen till Rust, but I needed _something_. 

Since you approved already, I'll let this one be and ping you in chat to see if we can come up with a better name